README.md

```diff
@@ -126,37 +131,39 @@ Refer to [consensus node documentation](docs/consensus.md).
 
 ## Smart contract development
 
-Please refer to [neo-go smart contract development
+Please refer to [NeoGo smart contract development
 workshop](https://github.com/nspcc-dev/neo-go-sc-wrkshp) that shows some
-simple contracts that can be compiled/deployed/run using neo-go compiler, SDK
+simple contracts that can be compiled/deployed/run using NeoGo compiler, SDK
```

Comment:
Not related to this line, but we also have `Usage:     "start a NEO node",` description for the `neo-go node` CLI command. Is it important?

Reply:
And `Usage: "create, open and manage a NEO wallet"` for `neo-go wallet` set of commands.

Reply:
And `Usage:     "convert addresses from existing NEO2 NEP6-wallet to NEO3 format",`.

Reply:
And `Usage:     "check and dump an existing NEO wallet",`.

Reply:
And `The neo-go compiler compiles Go programs to a bytecode that the NEO virtual machine can understand.` (this one is from documentation, so may be fixed in the current PR).

Reply:
And `Compiler translates interop function calls into NEO VM syscalls or (for custom functions) into NEO VM instructions.`.

Reply:
And `Magic number which uniquely identifies NEO network.`.

Reply:
And a couple of other NEO/Neo misuses.

Reply:
Patches are welcome, this one is about README mostly and the bug we have.

Reply:
OK, then I'll create a separate PR with these changes.